src/main/server/map/perlin-noise.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
+import { Canvas, createCanvas } from 'canvas';
+
+// Based on https://github.com/joeiddon/perlin/blob/master/perlin.js
+export class PerlinNoise {
+   generate(size: number): Canvas {
+      let canvas = createCanvas(size, size);
+      let ctx = canvas.getContext('2d');
+
+      let grid_size = size / 5;
+      let pixels = size;
```

Comment:
looks insane, but ok, maybe tidy it up later to prevent `ts-ignore`

Reply:
@dolanmiu Yeah its a huge pice of ... copied code from stackoverflow :D